index.html

```diff
@@ -75,134 +123,111 @@ <h2 class="works__card__title fw-700 fz-32 lh-44 clr-n-800">Tonic</h2>
+						  <li class="about__icons__li"><a href="#" class="about__icons__link link"><img src="Image/twitter.png" alt="twitter icon"></a></li>
+						  <li class="about__icons__li"><a href="#" class="about__icons__link link"><img src="Image/M.png" alt="M icon"></a></li>
+				      </ul>
+			      </div>
+				  <div class="about__btn">
+					  <button type="button" class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white">Get my resume</button>
+			      </div>
+		     </div>
+			 <ul class="about__tools ul">
+				 <li class="about__lang">
+					 <div class="about__lang__top flex">
+						 <h3 class="about__lang__top__title fw-500 fz-20 lh-24 clr-n-900">Languages</h3>
+						 <img class="about__lang__top__img" src="Image/arrow.png" alt="arrow icon">
+				      </div>
+					  <ul class="about__lang__bottom ul">
+						  <li class="flex bg-n-300"><img src="Image/js.png" alt="javascript icon"><p class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700">JavaScript</p></li>
+						  <li class="flex bg-n-300"><img src="Image/html.png" alt="html icon"><p class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700">HTML</p></li>
+						  <li class="flex bg-n-300"><img src="Image/css.png" alt="css icon"><p class="about__lang__bottom__name fw-500 fz-15 lh-20 clr-n-700">CSS</p></li>
+				      </ul>
+			      </li>
+				  <li class="about__frameworks">
+					  <div class="about__frameworks__top flex">
+						  <h3 class="about__frameworks__top__title fw-500 fz-20 lh-24 clr-n-900">Frameworks</h3>
+						  <img class="about__frameworks__top__img" src="Image/arrow.png" alt="arrow icon">
+					  </div>
+				  </li>
+				  <li class="about__skills">
+					  <div class="about__skills__top flex">
+						  <h3 class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900">Skills</h3>
+						  <img class="about__skills__top__img" src="Image/arrow.png" alt="arrow icon">
+						</div>
+			      </li>
+		      </ul>
+	      </section>
+		  
+		  <footer class="contact">
+			  <div class="contact__container">
+				  <h2 class="contact__title fw-700 fz-40 ">Contact me</h2>
+				  <p class="contact__dsc fw-500 fz-20 lh28">If you have an application you are interested in 
+					  developing, a feature that you need to build or a project 
+					  that needs coding.I would love to help with it.
+		          </p>
+				  <form action="https://formspree.io/f/xrgdbgjp" method="POST">
+					  <ul class="ul">
+						  <li>
+							  <input class="cntact__name fw-500 fz-17 lh-20" type="text" name="full-name" placeholder="juhar oumer seid">
+							</li>
+							<li>
+								<input type="email" name="email" placeholder="juharoumer@gmail.com" class="contact__email fw-500 fz-17 lh-20">
+				          </li>
+						  <li>
+							  <textarea class="contact__message fw-500 fz-17 lh-20" name="message" id="textarea" cols="30" rows="10" maxlength="500" placeholder="white your message here!"></textarea>
```

Comment:
- [ ] As demanded by the project requirement, please add a length of text that this input can take. You can use the maxlength="500" attribute. This will limit the user to writing a message of up to 500 characters.

- [ ] Kindly make sure that all input fields are required from the user before the form can be submitted. This will prevent the submission of half-filled forms. You can use the `required` attribute on all the form inputs and textarea.